.github/workflows/scripts/ping-codeowners-on-new-issue.sh

```diff
@@ -98,6 +107,9 @@ if [[ -n "${PING_LINES}" ]]; then
   # 3. The PING_LINES variable must be directly put into the printf string
   #    to get the newlines to render correctly, using string formatting
   #    causes the newlines to be interpreted literally.
+  echo "Pinging code owners"
```

Comment:
Would it make sense to print `PING_LINES` here?

Reply:
It would, good catch. I've updated this to print the mentions here.